mdbook/Preface.md

```diff
@@ -1,5 +1,12 @@
 # Preface
 
+This repo is my way of using the [Feynman Technique](https://medium.com/taking-note/learning-from-the-feynman-technique-5373014ad230) to learn Purescript and its ecosystem.
+
+Feynman used simple language, storytelling,
+comprehensively wrote down everything he knew about the topic,
+then attempted to teach it such that a child could understand it.
+Without jargon but with brevity, he identified what he didn't knew to the audience and had his content organized.
```

Comment:
```diff
- what he didn't knew to
+ what he didn't know to
```

Reply:
Could you put all of this one a single line? We'll let the tools handle the text wrapping here.

Reply:
Ok, I'll merge the lines. I use semantic linebreaks on my own texts so they're from there.

Reply:
Darn I missed the fix. A moment.

Reply:
That should be taken care of now.